gradle/test.gradle

```diff
@@ -144,60 +145,6 @@ subprojects {
     outputs.upToDateWhen { false }
   }
 
-  distributedTest {
-    useJUnit {
-      if (project.hasProperty("testCategory")) {
-        includeCategories += project.testCategory
-      }
-      excludeCategories 'org.apache.geode.test.junit.categories.FlakyTest'
```

Comment:
This is the exclusion to re-insert in the configure() closure. I believe you can do it externally as well only for distributed and performance, if you don't want to if-check the names inside the configure() loop